config/initializers/patch_vcr.rb

```diff
@@ -0,0 +1,28 @@
+# Problem with ruby 3.1.1 and VCR.
+# Should be patched, but no release yet.
+# This fixes it, and should probably be removed after next major VCR patch
+
+# frozen_string_literal: true
+
+if Rails.env.test?
```

Comment:
Kan ikke forstå at dette er nødvendigt - er det i det hele taget nødvendigt at have webmock med?

Reply:
Som jeg forstår det, skal VCR have et stubbing library til at håndtere erstatning af requests.

Reply:
Jeg leger lidt med det, og ser hvad jeg kan finde ud af.

Reply:
Uden WebMock kan jeg ikke kører tests offline med VCR, det kan jeg med WebMock.

Uden patch delen passer testen, men jeg får følgende fejl:
`bin/rails: warning: Exception in finalizer #<Proc:0x000000011550b188 /Users/claus/.rbenv/versions/3.1.1/lib/ruby/gems/3.1.0/gems/vcr-6.0.0/lib/vcr/library_hooks/webmock.rb:36 (lambda)>
/Users/claus/.rbenv/versions/3.1.1/lib/ruby/gems/3.1.0/gems/vcr-6.0.0/lib/vcr/library_hooks/webmock.rb:36:in `block in global_hook_disabled_requests': wrong number of arguments (given 1, expected 0) (ArgumentError)`
Jeg har ikke nok viden til at sige hvor stort et problem det er, og om patchen er nødvendig.

Reply:
Modtaget, ser ud til at der er et issue med nyere Ruby og nyere VCR - så vi lader det her være for nu :)